.github/actions/ktlint/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11.0.1-jdk-slim-stretch
+
+RUN apt-get update && apt-get -y install curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=5020686)